lib/resque/cluster/config.rb

```diff
@@ -62,10 +70,78 @@ def log_errors
 
       private
 
+      def host_maximums
+        case config_type
+        when :separate
```

Comment:
It didn't seem worthwhile when we're just going to turn around and rip it back out.